Apps/Admin/Client/Services/ICommunicationsApi.cs

```diff
@@ -32,29 +32,29 @@ public interface ICommunicationsApi
         /// <param name="communication">The model to add.</param>
         /// <returns>The wrapped model.</returns>
         [Post("/")]
-        Task<ApiResponse<RequestResult<Communication>>> Add([Body] Communication communication);
+        Task<RequestResult<Communication>> Add([Body] Communication communication);
```

Comment:
NIT: As a late entry I noticed that we should be renaming this to async methods

Reply:
Can fix.

Reply:
Actually, will leave for later since 2 of the 3 APIs already exist and haven't been updated to the naming convention yet.

Reply:
Sorry, I got quite confused and thought this was a comment on my PR.

Reply:
Changed.